.github/ISSUE_TEMPLATE/feature-request.yaml

```
@@ -0,0 +1,45 @@
+name: "💡 Feature Request"
+description: Request an ideas for this project
+title: "💡 [REQUEST] - <title>"
+labels: [
+  "Feature Proposal"
+]
```

I just made ```feature proposal``` label. So, please change label 😄 
```suggestion
labels: "feature proposal"
```